Week4/homework/ex1-aggregation/index.js

```diff
@@ -0,0 +1,98 @@
+import { MongoClient, ServerApiVersion } from "mongodb";
+import dotenv from "dotenv";
+dotenv.config();
+
+//Write a function that will return the array of the total population (M + F over all age groups) for a given Country per year.
+
+const helper = async (client,pipeline) =>{
+    const cursor = client 
+        .db("databaseWeek4")
+        .collection("population")
+        .aggregate(pipeline);
+
+    const res = await cursor.toArray(); 
+    const result = res.forEach(each => console.table(each));
+    return result;
```

Comment:
```suggestion
    return res.forEach(each => console.table(each));;
```

No need for a variable here

Reply:
Thanks, got it!